### PR TITLE
Convert

### PR DESCRIPTION
**Pull request by runboj**
1. use linkml to convert csv metadata file to json
2. update nmr_schema using slot such that we can do the first bullet point
3. reorganize the data folder 
4. minor updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,34 +47,35 @@ Schemasheets is part of the LinkML toolset that allows a LinkML data description
 Generation of spreadsheet through linkml schemasheets
 
 ```
-linkml2schemasheets-template -i nmr.yaml -o nmr_concise.tsv -s concise
+linkml2schemasheets-template -i src/nmr_schema/nmr_schema.yaml -o nmr_concise.tsv -s concise
 ```
 or
 ```
-linkml2schemasheets-template -i nmr.yaml -o nmr_exhaustive.tsv -s exhaustive
```
Comment (dylanmcreynolds) [inline README.md:54]: We should describe why you would do concise vs exhaustive

File: README.md [hunk 2]
```diff
@@ -47,34 +47,35 @@ Schemasheets is part of the LinkML toolset that allows a LinkML data description
...
+linkml2schemasheets-template -i src/nmr_schema/nmr_schema.yaml -o nmr_exhaustive.tsv -s exhaustive
 ```
 
 ### 2. Generate json schema
 
 Generating a corresponding (closed world) JSON-Schema definition (`nmr.schema.json`) for the NMR datasets:
```
Comment (dylanmcreynolds) [inline README.md]: What does `closed world` mean?

File: README.md [hunk 3]
```diff
@@ -47,34 +47,35 @@ Schemasheets is part of the LinkML toolset that allows a LinkML data description
...
 
 ```
-gen-json-schema --closed nmr.yaml  >nmr.schema.json
+gen-json-schema --closed src/nmr_schema/nmr_schema.yaml  >nmr.schema.json
 ```
 
 ### 3. Simple worked examples to validate the csv input metadata
```
Comment (dylanmcreynolds) [inline README.md]: `worked` should be `working`

File: README.md [hunk 4]
```diff
@@ -47,34 +47,35 @@ Schemasheets is part of the LinkML toolset that allows a LinkML data description
...
 
 Check ("valdate") that some csv metadata (`my-nmr-metadata.csv`) complies with the SFX metadata defintion (`nmr.yaml`):
 
 ```
-linkml-validate -s nmr.yaml my-nmr-metadata.csv
+linkml-validate -s src/nmr_schema/nmr_schema.yaml src/example_data/my-nmr-metadata.csv
```
Comment (dylanmcreynolds) [inline README.md]: This fails. I think it's because the `my-nmr-metadata.csv`, I don't have the csv file.

**Review by dylanmcreynolds — CHANGES_REQUESTED** [review]
A few documentaiton changes

File: README.md (hunk 2 above)
Comment (runboj) [reply README.md]: should be some copy paste typo. removed

File: README.md (hunk 3 above)
Comment (runboj) [reply README.md]: corrected

File: README.md (hunk 4 above)
Comment (runboj) [reply README.md]: I renamed the file, correcrted

File: README.md (hunk 1 above)
Comment (runboj) [reply README.md:54]: searched through the dococumentation, can't find where this part is mentiond...

**Pull request merged**